src/components/RestaurantOrder.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import RestaurantSumaButton from "./RestaurantSumaButton";
+import RestaurantRestaButton from "./RestaurantRestaButton";
+import "./RestaurantOrder.css";
+
+const RestaurantOrder = props => {
+  const pizzas = 0;
+  const [currentOrders, setOrders] = useState(pizzas);
+
+  function orderOne() {
+    setOrders(currentOrders + 1);
+  }
+
+  function reduceOne() {
+    if (currentOrders > 0) {
+      setOrders(currentOrders - 1);
```

Comment:
Good to see you considering and stopping the case for negative orders, it is always important to consider edge cases!